src/client/streaming.rs

```diff
@@ -40,11 +41,14 @@ where T: Message + Default,
 
         // Get the response
         let response = try_ready!(response);
+        let status_code = response.status();
 
         // Destructure into the head / body
         let (head, body) = response.into_parts();
 
-        if let Some(status) = super::check_grpc_status(&head.headers) {
```

Comment:
I'm just trying to be sure I understand, but it seems like the behavior *before* was that the existence of any `grpc-status` trailer would return an error, always? That seems like a woops...